bin/export_atlas_chebi_xml.pl

```diff
@@ -185,18 +191,49 @@ sub get_experiment_titles_from_db {
     # Experiment accessions are the keys of the new hash.
     my @expAccessions = keys %{ $expAccessions };
 
-	# Create Atlas database connection.
-	my $atlasDB = connect_pg_atlas;
 
     # Get the titles for the experiment accessions.
-    my $expAcc2title = $atlasDB->fetch_experiment_titles_from_atlasdb( \@expAccessions, $logger );
+    my $expAcc2title = fetch_experiment_titles_from_webpapi( \@expAccessions, $logger );
 
-    # Disconnect from Atlas DB.
-    $atlasDB->get_dbh->disconnect;
 
 	return $expAcc2title;
 }
 
+## fetch json formatted result for experiments and its titles from WebAPI.
+sub fetch_experiment_titles_from_webpapi {
+
+    my ( $accessions, $logger ) = @_;
+
+    my $url = "https://wwwdev.ebi.ac.uk/gxa/json/experiments";
+
+    my $json_hash;
+    my $expTitle;
+
+    # Empty hash for the results.
+    my $expAcc2title = {};
+
+   foreach my $expAcc  ( @{ $accessions } ) {
+        my $abs_url = join("/",$url,$expAcc);
+        my $ua = LWP::UserAgent->new;
+        my $response;
+        $response =  $ua->get($abs_url);
+        $logger->info( "Querying for experiment titles for $expAcc" );
+
+        if ($response->is_success) {
+            $json_hash = parse_json(decode ('UTF-8', $response->content));
+        }
+        else {
+         die $response->status_line;
```

Comment:
Here we probably want something more robust, like keep on going (to do the ones remaining) and keep track of the failed experiments, to later be able to either correct just those ones (maybe by re-running against an additional argument providing those ids). Again, if ChEBI linked experiments are only a few tenths, please neglect this comment.

Reply:
Thats a good suggestion. But as latter indicated Chebi ids linked experiments are few.